packages/data-transport-layer/src/services/run.ts

```diff
@@ -23,6 +29,9 @@ interface Bcfg {
     })
 
     const service = new L1DataTransportService({
+      nodeEnv: config.str('node-env', 'development'),
+      ethNetworkName: ethNetwork[config.str('eth-network-name')],
```

Comment:
Is the purpose of this to make it `undefined` if it is not one of the 3?

Reply:
yes and to make compiler pass with the stricter typing haha https://github.com/ethereum-optimism/optimism/pull/735#discussion_r626049932

Reply:
Maybe this should actually be chainid?

Reply:
We use chain id in the other services to denote which network they run on so I do think that we could replace network name with chain id. This will be useful because all networks have network ids and not all networks we may deploy on are named yet so we won't have to make any code changes if we use chain id